packages/corvid-cli/src/apps/pull.js

```diff
@@ -12,6 +12,9 @@ const pullApp = ({ useSsl = true, override = false, move = false } = {}) => ({
   serverMode: "clone",
   serverArgs: { override, move },
   handler: async (corvidConfig, win, client, localServerStatus) => {
+    logger.setExtraData("metasiteId", corvidConfig.metasiteId);
```

Comment:
logger.setExtraData("metasiteId", corvidConfig.metasiteId);
logger.setUserId(process.env.USER_GUID);
logger.setTag("command", process.env.COMMAND_CONTEXT);

the same code as in clone and open-editor
* maybe should be a util function
* maybe should be a function inside the logger that will set all 3 parameters together

Reply:
done